lava-v2-jobs-from-api.py

```diff
@@ -106,13 +106,18 @@ def get_job_params(config, test_config, defconfig, opts, build, plan):
     # hack for arm64 dtbs in subfolders
     if arch == 'arm64' and dtb:
         dtb = opts['dtb'] = os.path.basename(dtb)
-
-    job_name = '-'.join([
-        config.get('tree'), config.get('branch'), config.get('describe'),
-        arch, defconfig, dtb or 'no-dtb', device_type.name, plan])
-
-    url_px = '/'.join([
-        build['job'], build['git_branch'], build['kernel'], arch, defconfig])
+    if 'file_server_resource' in build:
```

Comment:
How about _always_ defining `file_server_resource` since it's created in `build.py` and simplifying this logic accordingly?

It won't be used when not publishing, but that should be harmless.

Reply:
I specifically made this script only use the file_server_resource information if the API provides it, so that it's backwards compatible for users outside of kernelci.org.

Reply:
OK - and `file_server_resource` isn't a mandatory field.  I guess this should be documented somewhere.